### DISCUSSION TITLE
Project still supported

### DISCUSSION THREAD

**Discussion by Travelbacon** [Foddy/node-red-contrib-huemagic#252] [General]
Dear @Foddy 

I noticed that this awesome project hasn't got any commit the last 14 months. I wander if this project is still alive? It would be very pity if this is not the case.

Cheers,
George.

**Comment by Foddy** [comment]
Sorry... I'm still behind this project and will bring out an update in the next days.

It was just a pretty eventful year with almost no time for nothing 😶

**Reply by Travelbacon** [reply]
That is very good to hear. :)

**Comment by Foddy** [comment]
Just wanted to let you guys know that I'm still working on the next HueMagic update. There are lots of changes with the new Philips Hue API under the hood (😏) but I guess that I'll be ready in approx. 3-7 days. Documenting new features and functions in 2 languages and so many different sections is really terrible! …

In the meantime, you might want to take a look here: https://github.com/Foddy/node-red-contrib-huemagic/discussions/254 :)

**Reply by Supergiovane** [reply]
I Understood. One of my nodes has 4 languages, one of wich "chinese". I loose more time to make documentation than to develop the node.

**Comment by Foddy** [comment]
The latest version is finally ready! :)
Let me know what you think!

**Comment by Travelbacon** [comment]
It works and I like the toggle button!
Is it an idea to add in the readme that with version 4 you have to reassign the bridge. And reassign the nodes again? My bulbs, sensors, and buttons started working again when I did this. ;)
